helm/deploy_tf_helm.sh

```diff
@@ -4,14 +4,21 @@ my_file="$(readlink -e "$0")"
 my_dir="$(dirname $my_file)"
 source "$my_dir/../common/common.sh"
 
-TF_HELM_URL=${TF_HELM_URL:-https://github.com/tungstenfabric/tf-helm-deployer/archive/master.tar.gz}
-wget $TF_HELM_URL -O contrail-helm-deployer.tar.gz
-mkdir -p contrail-helm-deployer
-tar xzf contrail-helm-deployer.tar.gz --strip-components=1 -C contrail-helm-deployer
-
-cd contrail-helm-deployer
+sudo mkdir -p /var/log/contrail
+TF_HELM_FOLDER=${TF_HELM_FOLDER:-tf-helm-deployer}
+TF_HELM_URL=${TF_HELM_URL:-https://github.com/tungstenfabric/tf-helm-deployer}
+if [ ! -d "$TF_HELM_FOLDER" ] ; then
+    git clone "$TF_HELM_URL" "$TF_HELM_FOLDER"
+fi
+cd tf-helm-deployer
 
 helm init --client-only
+#install plugin to make helm work without CNI
+if !( helm plugin list | grep -q tiller )
+then
+  helm plugin install https://github.com/rimusz/helm-tiller
+  helm tiller start &
```

Comment:
This task is forked. You should start only if it's not already started.. or kill "helm tiller" if it's running. Look at line 22 as an example. This command will fail if the port is already in use.